HyperPlay-Gaming/hyperplay-desktop-client
# [UX] Show Update button instead of Play for game with updates

Implements https://github.com/HyperPlay-Gaming/product-management/issues/51
Changed the previous approach now as discussed with the team yesterday. Now HP is showing an update button on the game page as well.

![image](https://github.com/HyperPlay-Gaming/hyperplay-desktop-client/assets/26871415/aab45123-63e5-4c2e-bf30-b0778d40de6a)


On another PR I will fix the behavior when launching games with update from the store.
Right now it does not check for update in this case.


### To Test
- Find a HyperPlay game with an update (or manually edit the `hp_store/library.json` file to lower the version of an installed game and hit the refresh button on the library)
- Then check if the Play button changed to an Update button that will trigger the update.



---

Use the following Checklist if you have changed something on the Backend or Frontend:
